### PR TITLE
Upgrade to VS 2022

### PR DESCRIPTION
**Pull request by SameerK-MSFT**
Fixes #
Maui in DevOps

**Changes proposed in this request**
Update AndroidHandler as required
Remove condition for MAUI
Update build template

**Testing**
Existing unit, integration, tests 

**Performance impact**
None

### CODE REVIEW

File: build/template-OneBranch-libsandsamples.yaml [hunk 1]
```diff
@@ -20,14 +20,22 @@ steps:
...
     solution: ${{ parameters.Solution }}
+    vsVersion: 17.0
     msbuildArgs: '/t:restore'
     platform: ${{ parameters.BuildPlatform }}
     configuration: ${{ parameters.BuildConfiguration }}
 
+- template: /SharedComponents/OneBranch/Steps/Prepare.MSBuild.yml@templates
+  parameters:
+    msBuildVersion: 17.0
+    msbuildInstallationPath: c:\.vs
```
Comment (bgavrilMS) [inline build/template-OneBranch-libsandsamples.yaml:31]: does this need to be hardcoded?

**Review by bgavrilMS — APPROVED** [review]

File: build/template-OneBranch-libsandsamples.yaml (hunk 1 above)
Comment (trwalke) [reply build/template-OneBranch-libsandsamples.yaml]: no, it can be a variable that is settable when stating the build

**Comment by bgavrilMS** [comment]
@SameerK-MSFT - is this still relevant?

**Comment by SameerK-MSFT** [comment]
The work has been done elsewhere. Closing this.

**Pull request closed without merging**